4/Dockerfile

```diff
@@ -17,7 +17,8 @@ RUN set -ex \
   ; do \
     gpg --keyserver hkp://pgp.mit.edu:80 --recv-keys "$key" || \
     gpg --keyserver hkp://keyserver.pgp.com:80 --recv-keys "$key" || \
-    gpg --keyserver hkp://ha.pool.sks-keyservers.net:80 --recv-keys "$key" ; \
+    gpg --keyserver hkp://ipv4.pool.sks-keyservers.net:80 --recv-keys "$key" || \
```

Comment:
Again, servers in the ipv4 pool are not guaranteed to support port 80 -- the port here should be removed.

Reply:
Ok